packages/block-editor/src/components/media-replace-flow/README.md

```diff
@@ -0,0 +1,74 @@
+# Media replace flow
+
+A component that implements a replace flow for various media. It is used to allow various blocks that use media to have a toolbar button for replacing it. I offers several options, such as:
```

Comment:
Maybe "A component that implements a replace flow for various media" -> "A component that implements a replacement flow for various media objects".